Various manual page fixes. Selected from #668524 and #668461.

Patch was sent upstream:
http://lists.gnu.org/archive/html/bug-groff/2011-01/msg00011.html

diff -uNPrp a/contrib/mm/groff_mm.man b/contrib/mm/groff_mm.man
--- a/contrib/mm/groff_mm.man	2011-01-21 14:52:52.302854667 +0100
+++ b/contrib/mm/groff_mm.man	2011-01-21 15:51:53.138775191 +0100
@@ -1158,7 +1158,7 @@ and also less than or equal to
 .B Hb
 or
 .B Hs
-are centerered.
+are centered.
 .RE
 .
 .IP
@@ -1229,7 +1229,7 @@ Fourteen number registers named
 up to
 .B H14
 contain the counter for each heading level.
-The values are printed using arabic numerals;
+The values are printed using Arabic numerals;
 this can be changed with the macro
 .B HM
 (see below).
@@ -1590,7 +1590,7 @@ is still controlled by
 .IR text-indent .
 .
 .IP
-The mark is left-justified whitin the mark area if
+The mark is left-justified within the mark area if
 .I pad
 is\~0.
 If
@@ -1664,7 +1664,7 @@ Default is\~0.
 List-status clear.
 Terminates all current active lists down to
 .IR list-level ,
-or\~0 if no argmuent is given.
+or\~0 if no argument is given.
 This is used by\~\c
 .B H
 to clear any active list.
diff -uNPrp groff-1.21.old/man/roff.man groff-1.21/man/roff.man
--- groff-1.21.old/man/roff.man	2010-12-31 08:33:09.000000000 +0100
+++ groff-1.21/man/roff.man	2011-01-21 15:26:24.061849971 +0100
@@ -252,7 +252,7 @@ Today, the name
 .I roff
 is used to refer to a
 .I troff/\:nroff
-sytem as a whole.
+system as a whole.
 .
 .P
 Ossanna's first version was written in the PDP-11 assembly language and
